index.html

```diff
@@ -0,0 +1,48 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Prangon's Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <header>
+        <div class="container">
+            <div class="logo-placeholder">
+                <div class="my-logo">
+                    PG
+                </div>
+            </div>
+            <div class="menu">
+                <div class="icon">
+                    <div class="union">
+                        <button class="nav-button"><img src="images/Union.png" alt="menu-image" class="menu-image"></button>
+                    </div>
+                </div>
+            </div>
+        </div>
+    </header>
+    <div class="container">
+        <div class="headline">
+            <div class="primary-text">
+                I'm Prangon<br>Glad to see you!
+            </div>
```

Comment:
- I think it will be better to replace your entire ` <div class="primary-text">` to a `<h1>` tag.  You can do this instead `<h1 class="primary-text">`👍🏽